README.md

```diff
@@ -83,7 +85,7 @@ Options can be provided via (in order of precedence) the programmatic API, the C
 |:----------------|:---------------------|:-------------------------------|:------------
 | `--target -t`   | -                    | Depends.                       | One or more targets\*
 | `--all -a`      | -                    | `false`                        | Build all known targets.<br>Takes precedence over `--target`.
-| `--napi`        | -                    | `false`                        | Make [N-API][n-api] build(s).<br>Targets default to node and electron.
+| `--napi`        | -                    | `false`                        | Make [N-API][n-api] build(s).<br>Targets default to latest node and electron, which can be overridden with `--all` or `--target`.
```

Comment:
```suggestion
| `--napi`        | -                    | `false`                        | Make [N-API][n-api] build(s).<br>Targets default to latest node and electron, which can be overridden with `--target`. Note: `--all` should be avoided for now because it includes targets that don't support N-API.
```